model-engine/model_engine_server/infra/gateways/live_model_endpoint_infra_gateway.py

```diff
@@ -151,6 +153,8 @@ async def update_model_endpoint_infra(
             gpu_type = infra_state.resource_state.gpu_type
         if storage is None:
             storage = infra_state.resource_state.storage
+        # Don't allow changing nodes_per_worker
```

Comment:
should we error if nodes_per_worker is not equal to what the infra is set to?

Reply:
ah so I made the api such that you cannot change nodes_per_worker (changing between lws and deployment seems too annoying)